lib/providers/controllerAutoCompleteProvider.js

```diff
@@ -283,6 +282,65 @@ const customAlloyCompletionRules = [
 		}
 	},
 
+	{
+		regExp: /(?:Alloy)?\S+/i,
```

Comment:
Do we need the non-capturing group and quantifiers here? Would `/Alloy/` not be simple enough to catch it?